### DISCUSSION TITLE
How to use issuebot？

### DISCUSSION THREAD

**Discussion by ghost** [community/community#68091] [Programming Help]
How to use issubot to automatically open a pull request and add files to a new branch?

**Comment by milindaShehan** [comment]
To use issubot to automatically open a pull request and add files to a new branch, you can use the following steps:

1. Create a new branch on your repository.
2. Add the files that you want to add to the pull request to the new branch.
3. Commit the files to the new branch.
4. Create a new issue on your repository.
5. In the issue description, add the following text:

`/issubot:pr`

6.  Save the issue.

Issubot will then automatically create a pull request for you, adding the files from the new branch.

You can also use issubot to specify the branch that you want to create the pull request from and the branch that you want to merge the pull request into. To do this, add the following text to the issue description:

```
/issubot:pr
branch: [branch_name]
merge_branch: [branch_name]
```
For example, to create a pull request from the new-feature branch and merge it into the main branch, you would add the following text to the issue description:

```
/issubot:pr
branch: new-feature
merge_branch: main
```
Issubot is a powerful tool that can help you to automate the process of creating and merging pull requests. By using issubot, you can save time and effort, and ensure that your pull requests are always up-to-date.

Here is an example of how to use issubot to automatically open a pull request and add files to a new branch:
```
Create a new branch
git checkout -b new-feature

Add the files that you want to add to the pull request to the new branch
git add new-file.txt

 Commit the files to the new branch
git commit -m "Added new file"

 Create a new issue on your repository
```
In the issue description, add the following text:

`/issubot:pr`
Save the issue.

Issubot will then automatically create a pull request for you, adding the files from the `new-feature` branch.

You can then review the pull request and merge it into the main branch.
